lisp/ess-custom.el

```diff
@@ -887,7 +887,7 @@ See `ess-style-alist' for for an overview of ESS indentation."
      (ess-indent-function-declaration . t)
      (ess-indent-from-lhs . t)
      (ess-indent-with-fancy-comments . t))
-    
```

Comment:
>>> Lionel Henry on Mon, 01 Jun 2015 14:10:18 -0700 wrote:

 LH> this mode looks very nice. 

Yes ... only if it wasn't buggy. I have already opened two issues for it.